pkg/common/utils/utils_test.go

```diff
@@ -144,7 +144,7 @@ func TestQuerySnapshotsUtil(t *testing.T) {
 	// Create context
 	commonUtilsTestInstance := getCommonUtilsTest(t)
 
-	volumeManager := cnsvolumes.GetManager(ctx, commonUtilsTestInstance.vcenter, nil, false)
+	volumeManager := cnsvolumes.GetManager(ctx, commonUtilsTestInstance.vcenter, nil, false, false)
```

Comment:
The tests could read this off a feature flag, this way when you enable the feature, all unit tests will be automatically run, and you will not need to go back to change these param to `true`

Reply:
we are planning to update unit tests for the snapshot feature when we implement the snapshot feature for multi vCenter. we will consider your comment while updating unit tests.